features/cli/cli_enable.feature

```diff
@@ -0,0 +1,631 @@
+        "_schema_version": "0.1",
+        "errors": [],
+        "failed_services": [],
+        "needs_reboot": false,
+        "processed_services": [
+          "esm-infra"
+        ],
+        "result": "success",
+        "warnings": []
+      }
+      """
+    When I run `pro disable esm-infra` with sudo
+    And I verify that running `pro enable esm-infra foobar --format json --assume-yes` `with sudo` exits `1`
+    Then stdout is a json matching the `ua_operation` schema
+    And API full output matches regexp:
+      """
+      {
+        "_schema_version": "0.1",
+        "errors": [
+          {
+            "additional_info": {
+              "invalid_service": "foobar",
+              "operation": "enable",
+              "service_msg": "Try <valid_services>"
+            },
+            "message": "Cannot enable unknown service 'foobar'.\nTry <valid_services>",
+            "message_code": "invalid-service-or-failure",
+            "service": null,
+            "type": "system"
+          }
+        ],
+        "failed_services": [
+          "foobar"
+        ],
+        "needs_reboot": false,
+        "processed_services": [
+          "esm-infra"
+        ],
+        "result": "failure",
+        "warnings": []
+      }
+      """
+    When I run `pro disable esm-infra esm-apps` with sudo
+    And I run `pro enable esm-infra esm-apps --beta --format json --assume-yes` with sudo
+    Then stdout is a json matching the `ua_operation` schema
+    And API full output matches regexp:
+      """
+      {
+        "_schema_version": "0.1",
+        "errors": [],
+        "failed_services": [],
+        "needs_reboot": false,
+        "processed_services": [
+          "esm-apps",
+          "esm-infra"
+        ],
+        "result": "success",
+        "warnings": []
+      }
+      """
+
+    Examples: ubuntu release
+      | release | machine_type  | valid_services                                                                                                                             |
+      | xenial  | lxd-container | anbox-cloud, cc-eal, cis, esm-apps, esm-infra, fips, fips-preview,\nfips-updates, landscape, livepatch, realtime-kernel, ros, ros-updates. |
+      | bionic  | lxd-container | anbox-cloud, cc-eal, cis, esm-apps, esm-infra, fips, fips-preview,\nfips-updates, landscape, livepatch, realtime-kernel, ros, ros-updates. |
+      | focal   | lxd-container | anbox-cloud, cc-eal, esm-apps, esm-infra, fips, fips-preview, fips-updates,\nlandscape, livepatch, realtime-kernel, ros, ros-updates, usg. |
+      | jammy   | lxd-container | anbox-cloud, cc-eal, esm-apps, esm-infra, fips, fips-preview, fips-updates,\nlandscape, livepatch, realtime-kernel, ros, ros-updates, usg. |
+      | noble   | lxd-container | anbox-cloud, cc-eal, esm-apps, esm-infra, fips, fips-preview, fips-updates,\nlandscape, livepatch, realtime-kernel, ros, ros-updates, usg. |
+
+  Scenario Outline: Attached enable of ESM services in a ubuntu machine
```

Comment:
Question - should tests like this have their own `esm.feature` file, leaving this one only testing the generic enable details? (fine if not)

Reply:
I think it probably should, but I want to get this already massive PR in and it's already a great improvement.